main.py

```diff
@@ -366,22 +504,24 @@ async def on_ready():
     for key in linkQueue: #for every channel
       for link in linkQueue[key]: #for every link in the queue
         running = False #default to not open
-        try:
-          driver.get(link[1]) #go to link
-          print("loading "+link[1])
-          await asyncio.sleep(1)
-          #it's open if the link went through
-          running = ("Join" in html2text.HTML2Text().handle(driver.page_source)) and ("meet" in driver.current_url)
-        except: #if timeout, due to bot detection
-          running = False #default to not open
-          driver.close() #restart driver
-          await startDriver()
-          print("ohhh nooo")
+        if "meet.google.com" in link[1]:
+          try:
+            driver.get(link[1]) #go to link
+            print("loading "+link[1])
+            await asyncio.sleep(1)
+            #it's open if the link went through
+            running = ("Join" in html2text.HTML2Text().handle(driver.page_source)) and ("meet" in driver.current_url)
+          except: #if timeout, due to bot detection
+            running = False #default to not open
+            driver.close() #restart driver
+            await startDriver()
+            print("ohhh nooo")
+        else:
+          running = True
         print(running)
         if running:
           dropLinks[key].append(link)
-          await msgCh("<:pingo:822111531063836712> <@&"+str(link[0])+"> "+link[2]+" is online now !1!!\nBreak is Over! Stop playing games! Stop watching youtube!\n<"+link[1]+">",int(key))
-          await msgCh("<:blushylakshy:814288474010025994>",int(key))
+          await msgCh(f"<:pingo:822111531063836712> <@&{link[0]}>, {link[2]} is now online at <{link[1]}>",int(key))
```

Comment:
can you change all the user feedback so that the bot's messages are not different?

Reply:
I left you a pingo man

Reply:
In any case, as a bot cannot use emoji from servers it is not part in, this significantly reduces portability if custom emoji are used, which is not quite in line with claims on the README or negatively impacts those who wish to run the software themselves.

Reply:
Would dropping all emoji but retaining older messages do?

Reply:
> this significantly reduces portability

I don't care keep the emojis and don't change the messages >:(

Reply:
Unfortunately, this is one area I am not willing to budge in. The pingos must die.